src/routes/components/RedirectLegacyPath/RedirectLegacyPath.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { memo, VFC } from 'react';
+import { Navigate } from 'react-router';
+import { useRouteParams } from '../../../hooks';
+
+interface Props {
+
+  /**
+   * New route root path.
+   * For example: `to="producer"`.
+   */
+  readonly to: string;
+}
+
+/**
+ * Redirects to given path with `id` property.
+ * Serves as `vndb` legacy path handler.
+ */
+const RedirectLegacyPathComponent: VFC<Props> = ({ to }) => {
+  const { id } = useRouteParams<{ id: string; }>();
```

Comment:
What if id isn't valid, like `/idfoobar`? You could add 404 error handling in this PR 😺

Reply:
Well, it would be great if we had design for 404 page 🙃

Reply:
done https://www.figma.com/file/YXz9Xy3xEA3C4G45DGa9cc/vndb?node-id=1262%3A29518

Reply:
As for `404` error from server it's a bit complicated. It will be handled by `ContentWrapper` but I've added placeholder for non-existent routes